eclipse/src/saros/ui/util/CollaborationUtils.java

```diff
@@ -273,24 +268,23 @@ public void run() {
    */
   private static String getSessionDescription(ISarosSession sarosSession) {
 
-    final Set<IReferencePoint> projects = sarosSession.getReferencePoints();
+    final Set<IReferencePoint> referencePoints = sarosSession.getReferencePoints();
 
     final StringBuilder result = new StringBuilder();
 
-    for (IReferencePoint project : projects) {
+    for (IReferencePoint referencePoint : referencePoints) {
 
       final Pair<Long, Long> fileCountAndSize;
 
       final List<IResource> resources =
-          Collections.singletonList(ResourceConverter.getDelegate(project));
+          Collections.singletonList(ResourceConverter.getDelegate(referencePoint));
 
       fileCountAndSize = FileUtils.getFileCountAndSize(resources, true, IContainer.EXCLUDE_DERIVED);
 
       result.append(
           String.format(
-              "\nProject: %s (%s), Files: %d, Size: %s",
-              project.getName(),
-              "complete",
+              "\nReference Point: %s, Files: %d, Size: %s",
+              referencePoint.getName(),
```

Comment:
Does this return the full path or just the folder name ?

XYZ invited you to:

Reference Point: src, Files: 5 ...
Reference Point: src, Files: 25 ...
Reference Point: src, Files: 35 ...
Reference Point: src, Files: 7 ...
Reference Point: src, Files: 4711 ...

I do not think this is very helpful

Reply:
I can unpack the reference point and call `getFullPath()` on the delegate if you would prefer. This could, however, lead to very long paths being displayed in the UI, making it very cluttered. In general, this is the same issue as #1042.

Reply:
Done.